File: TeeNew/Builder2007/Assign_Picture.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Assign_Picture.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma link "TeePolar"
#pragma resource "*.dfm"
TAssignPicture *AssignPicture;
//---------------------------------------------------------------------------
__fastcall TAssignPicture::TAssignPicture(TComponent* Owner)
        : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TAssignPicture::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues(8);
}
//---------------------------------------------------------------------------

void __fastcall TAssignPicture::Button1Click(TObject *Sender)
{
  Image1->Picture->Assign(Chart1);
}
//---------------------------------------------------------------------------

File: TeeNew/Assign_Picture.pas
unit Assign_Picture;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  Base, TeEngine, Series, TeePolar, TeeProcs, Chart;

type
  TAssignPicture = class(TBaseForm)
    Button1: TButton;
    Series1: TRadarSeries;
    Image1: TImage;
    Splitter1: TSplitter;
    procedure FormCreate(Sender: TObject);
    procedure Button1Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TAssignPicture.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues(8);
end;

procedure TAssignPicture.Button1Click(Sender: TObject);
begin
  Image1.Picture.Assign(Chart1);
end;

initialization
  RegisterClass(TAssignPicture);
end.
